vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -199,45 +200,49 @@ public Retrofit getRetrofit() {
         return mRetrofit;
     }
 
+    /**
+     * @return the current VimeoAccount, or null if it hasn't been set yet
+     */
+    @Nullable
     public VimeoAccount getVimeoAccount() {
-        if (mVimeoAccount == null) {
-            throw new AssertionError("Account should never be null");
-        }
-
         return mVimeoAccount;
```

Comment:
It seems potentially dangerous to return the reference to the actual private member. Is this a safe move or should it be cloned?